src/sentry/static/sentry/app/components/events/eventCauseEmpty.jsx

```diff
@@ -0,0 +1,192 @@
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import * as Sentry from '@sentry/browser';
+
+import Button from 'app/components/button';
+import CommitRow from 'app/components/commitRow';
+import {Panel} from 'app/components/panels';
+import {promptsUpdate} from 'app/actionCreators/prompts';
+import robotLogo from 'app/../images/robot-logo.png';
+import SentryTypes from 'app/sentryTypes';
+import {snoozedDays} from 'app/utils/promptsActivity';
+import space from 'app/styles/space';
+import {t} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
+import withApi from 'app/utils/withApi';
+
+const DUMMY_COMMIT = {
+  id: 'ff721e0',
+  author: {name: 'Uh/Oh'},
```

Comment:
The icon character isn't Uh/Oh though :(

I don't know if he has a name. /cc @cameronmcefee for approval of this anyway

Reply:
Yeah, this either needs to be a different icon or different name. How about this one and then some some terrible cat name like "Codesworth"

![cat](https://user-images.githubusercontent.com/72919/61087978-000d6d00-a3ec-11e9-88a1-f3e2697d4893.png)

Reply:
wow I love `codesworth`

Reply:
haha yes! thanks @cameronmcefee